Remove unused exports from Browser System.Native build

marek-safar opened the pull request:
*This is waiting on few more cleanups on libraries side*

marek-safar committed:
Remove unused exports from Browser System.Native build

wfurt reviewed (approved):
LGTM. There were some wasm errors but they look unrelated. You better look @marek-safar

marek-safar committed:
Cleanup SPC Unix specific files

marek-safar committed:
Add Browser specific PersistedFiles implementation

Simplified the implementation removes dependency on several native calls
which all return only default values.

marek-safar committed:
Move more files to Unix section

marek-safar committed:
Move GetCurrentProcessId for Browser to managed

It returns constant all the time

marek-safar committed:
Resolve conflicts

vargaz commented:
The linker is supposed to link out unused functions.
